### PR TITLE
fix(web-widgets): add public-hoist patterns

### PR DESCRIPTION
**Pull request by iobuhov**
### Pull request type

Bug fix (non-breaking change which fixes an issue)

---

### Description
With `pnpm` update to 7.29 our PWT jest config don't work anymore (which is expected, as we using [node-linker=isolated](https://pnpm.io/npmrc#node-linker)). To fix that, we need to mark some deps as "public" using [public-hoist-pattern](https://pnpm.io/npmrc#public-hoist-pattern). If package match pattern it will be available in `node_modules` even if it's not listed in package dependencies.

### CODE REVIEW

File: packages/customWidgets/signature-web/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "@types/react-resize-detector": "^5.0.0",
     "css-loader": "^6.7.3",
     "eslint": "^7.32.0",
+    "identity-obj-proxy": "^3.0.0",
```
Comment (leonardomendix) [inline packages/customWidgets/signature-web/package.json:47]: I didn't get why we need that.

Comment (iobuhov) [reply packages/customWidgets/signature-web/package.json:47]: This packages is part of jest config, `identity-obj-proxy` used as a mock for `CSS` and `SCSS` modules.

Comment (iobuhov) [reply packages/customWidgets/signature-web/package.json:47]: And it was missing in package deps

**Review by r0b1n — APPROVED** [review]

File: packages/customWidgets/signature-web/package.json (hunk 1 above)
Comment (leonardomendix) [reply packages/customWidgets/signature-web/package.json:47]: Thanks!

**Review by leonardomendix — DISMISSED** [review]

**Review by leonardomendix — APPROVED** [review]

**Pull request merged**